README.md

```diff
@@ -4,21 +4,36 @@
 
 [![](https://img.shields.io/travis/aashutoshrathi/Testcase-Generator?style=for-the-badge)](https://travis-ci.org/aashutoshrathi/Testcase-Generator)
 
-One Click Test Case Generation for HackerRank Problems.
+One Click Test Case Generation for [HackerRank](https://eee.hackerrank.com), [HackerEarth](https://www.hackerearth.com) and [CodeChef](https://www.codechef.com) Problems.
```

Comment:
What is the difference between `eee` and `www`?

Reply:
Fixed typo :sweat_smile: